string object_frame_id
string parent_frame_id
string arm
float64 xlength
float64 ylength
---
bool success
float64 box_lx
float64 box_ly
float64 box_lz
float64[] position
float64[] rotation